### ISSUE TITLE
Enhance DND/mute behaviour

### ISSUE THREAD

**Issue by philipp2310** [project-alice-assistant/ProjectAlice#348]
WIP WIP WIP

This is a collection of ideas for enhancing the DND status and the mute functionality.

Ways Alice can talk and when it should be muted:
**Never**
- "Alarm" -> "I detect smoke in the living room!" 

**Soft DND (phone call active, TV running)**
- "Random chatter" / small talk
- "System Information" -> "I reconnected to the Internet"

**DND**
- "Notification", e.g. Mail -> "You got a new message from psycho"

**microphone off**
- wake word detected -> "Yes?"

Apart from "wake word off", everything is using .say() functionality.
[ ] The skill must provide a "level" e.g. chatter/system/notification/alarm.. and say() looks at the current status of the target DEVICE
- + easy to develop
[ ] The skill provides its name, a central authorization check determines if that skill is allowed to speak at the moment.
- + highly flexible
[ ] combination of both above, e.g. limit the "level" a skill can use

**Comment by LazzaAU** [comment]
Would this also be a good spot to add a "location" include/exclude list . ?

EG: play this chatter notification type in all the rooms but not in the kids room 
or
if its between this time frame play this notification here but not there

**Comment by philipp2310** [comment]
yep, should be included as well - and that is where it gets complicated as it is adding even one more vector to configure ^^
